contents/templates/ces-survey/index.mdx

```diff
@@ -0,0 +1,51 @@
+---
+title: Customer effort score (CES) survey
+subtitle: How easy is it to use your product? 
+thumbnail: ../thumbnails/ces-survey.png
+featuredImage: ../thumbnails/featured/ces-survey.png
+showTitle: true
+filters: {
+    type: ["product","survey"],
+    maintainer: official
+}
+---
+
+<Section
+    divider={false}
+    cols={2}
+>
+    <div>
+        <h5>Effort rating scale</h5>
+        <p>Users rate the ease of their experience from 1 (very difficult) to 5 (very easy).</p>
+    </div>
+    <div>
+        <h5>Understand the usability of your product</h5>
+        <p>Provides insight into how challenging your product is at a glance.</p>
+    </div>
+</Section>
+
+<section class="my-12 px-5">
+  <div class="max-w-screen-2xl mx-auto">
+    <h3 class="text-2xl lg:text-3xl m-0 text-center mb-6 sm:mb-12">What's good about this survey?</h3>
+    <div class="flex justify-center">
+      <ul class="m-0 p-0 list-none inline-grid sm:grid-cols-2 justify-evenly relative after:absolute after:w-full after:left-0 after:top-1/2 after:-translateY-1/2 before:absolute before:h-full before:top-0 before:left-1/2 after:-translateX-1/2 sm:after:block after:hidden sm:before:block before:hidden">
+        <li class="relative md:max-w-md py-4 md:py-8 pl-4 pr-2 md:px-12">
+          <h5 class="text-xl font-extrabold m-0 pb-1 pr-4">Instant clarity</h5>
+          <p class="m-0 text-[15px]">CES provides immediate insights into areas that may need simplification or improvement.</p>
+        </li>
+        <li class="relative md:max-w-md py-4 md:py-8 pl-4 pr-2 md:px-12">
+          <h5 class="text-xl font-extrabold m-0 pb-1 pr-4">Focus on usability</h5>
+          <p class="m-0 text-[15px]">Emphasizes the importance of user-friendly experiences, ensuring customer needs are met.</p>
+        </li>
+        <li class="relative md:max-w-md py-4 md:py-8 pl-4 pr-2 md:px-12">
+          <h5 class="text-xl font-extrabold m-0 pb-1 pr-4">Areas for improvement</h5>
+          <p class="m-0 text-[15px]">Running this survey on different pages gives an idea of what areas are most difficult to use.</p>
+        </li>
+        <li class="relative md:max-w-md py-4 md:py-8 pl-4 pr-2 md:px-12">
+          <h5 class="text-xl font-extrabold m-0 pb-1 pr-4">Customer-centric approach</h5>
+          <p class="m-0 text-[15px]">Demonstrates the product's commitment to reducing customer effort and enhancing satisfaction.</p>
```

Comment:
This is true broadly across all the surveys, but I think we could change these sections to be more valuable to users by answering common questions. 

E.g. call the section 'Getting started with CES surveys' and make the points address this area. Otherwise, we risk repeating info over and over. 

1. What's a good CES score? CES measures how difficult your product is to use on a 1-5 scale. Higher is better, and 4+ is considered a strong result. 
2. Why is CES important? [Studies](https://hbr.org/2010/07/stop-trying-to-delight-your-customers) have shownCES is a better indicator of customer loyalty than many other metrics and correlates to retention.
3. Tips for using CES surveys: CES surveys reveal ease of use, but not overall value. They should be used alongside NPS surveys to get the full picture.
4. When to use CES surveys: CES surveys are especially useful for new users, and work great when deployed after onboarding flows. 

Nothing wrong with the content you have, but I feel this approach if used throughout would start to offer more value.

Reply:
If we can answer common follow up questions on these pages, we'll really be adding value. Everyone will immediately want to know benchmarks, why it's important, and best practices.

Reply:
Got, rewrote each of them along these lines.